tests/test_name.py

```diff
@@ -467,3 +468,779 @@ def test_first_name_initials_without_whitespace_is_initialized_correctly():
 
 def test_last_name_recognized_correctly_regression_test():
     assert u"De Sousa Vieira" == ParsedName.loads('De Sousa Vieira, M.C.').last
+
+
+def test_elastic_search_visitor_author_lastname_firstname_with_commas_and_initials():
```

Comment:
```suggestion
def test_generate_es_query_lastname_firstname_with_commas_and_initials():
```
and same for all other tests.